test/sync_vsc_email.py

```diff
@@ -81,9 +81,8 @@ def test_sync(self, pl):
         newmap = [
             "vsc40001@vscentrum.be huppel@bar.com",
             "vsc40002@vscentrum.be foo2@bar.com",
-            "",
         ]
 
         vps.do(False)
         with open(cmfn, 'r') as c:
-            self.assertEqual(c.read(), "\n".join(newmap))
+            self.assertEqual(c.readlines(), newmap)
```

Comment:
readlines does not strip the newline char (i think)

Reply:
indeed. Fixed it :)